client/components/search/index.jsx

```diff
@@ -166,7 +166,9 @@ const Search = React.createClass( {
 
 	scrollOverlay: function() {
 		window.requestAnimationFrame( () => {
-			this.refs.overlay.scrollLeft = this.refs.searchInput.scrollLeft;
+			if ( this.refs.overlay && this.refs.searchInput ) {
```

Comment:
I've kept the check, in case anything changes before the callback, and added another before requesting the frame.